sdks/java/extensions/sql/zetasql/src/test/java/org/apache/beam/sdk/extensions/sql/zetasql/ZetaSQLDialectSpecTest.java

```diff
@@ -837,7 +834,6 @@ public void testLikeAllowsEscapingBackslash() {
   }
 
   @Test
-  @Ignore("Currently non UTF-8 values are coerced to UTF-8")
   public void testThrowsErrorForNonUTF8() {
```

Comment:
This test name doesn't seem correct after these changes. What is it testing now? What about the previous assertions on throwing an exception for invalid utf8?

Reply:
This is a good point. I will choose to remove this test. I tried to dig into ZetaSQL's documentation and internal tests suite, there was no clear explanation how to deal with nonUTF8 chars by LIKE operator (there is are code-generated nonUTF8 test cases, but they don't have clear comments to demonstrate their purpose) 

So I will leave this part to be tested by internal test suite.

Reply:
PR updated.